Exercise_2.java

```diff
@@ -12,14 +14,47 @@ void swap(int arr[],int i,int j){
     
     int partition(int arr[], int low, int high) 
     { 
-   	//Write code here for Partition and Swap 
+    	int pivot = arr[high];  

```

Comment:
Good Job! Please add time complexity and space complexity and comments about your implementation.